components/curators-list.tsx

```diff
@@ -39,7 +39,7 @@ export default function CuratorsList() {
   const { state, dispatch } = useAppState();
 
   function handlerSearch() {
-    if (!state.Service?.network?.active) return;
+    if (!state.Service?.network?.active && inView === false) return;
```

Comment:
I think that the correct condition is `!state.Service?.network?.active || inView === false`, because when `state.Service?.network?.active` is `true` and `inView` is `false` it won't return